src/app/modules/entry/views/entry-preview/entry-preview.component.ts

```diff
@@ -93,11 +93,15 @@ export class EntryPreviewComponent extends EntryBase implements OnInit {
       },
       tooltip : {
         formatter: params => {
-          const { value } = Array.isArray(params) ? params[0] : params;
+          const { value, dataIndex } = Array.isArray(params) ? params[0] : params;
+          const progressValue = ReportHelper.time(String((dataIndex + 1) / 100 * this._duration));
```

Comment:
@diamond-darrell - try using ReportHelper.time(String((dataIndex) / 99 * this._duration)); 
Check short and long videos and see if this works for all.

Reply:
> @diamond-darrell - try using ReportHelper.time(String((dataIndex) / 99 * this._duration));
> Check short and long videos and see if this works for all.

why 99? 🙂

Reply:
ok, it's mostly fine, threre might be some edge cases, but we probably can say it's a limitation